effectsize_example.Rmd

```diff
@@ -85,7 +100,7 @@ library(ggstance)   # for geom_pointrangeh()
 
 ```{r boilerplate, include = FALSE}
 set.seed(12)
-format_num <- function(...) format(..., digits = 2)
+format_num <- function(x, k = 2) format(round(x, k), nsmall = k)
```

Comment:
I'm confused. Is the above output on the F values with 5 significant digits? Why not the 3 as per the format_num function, so you would get 416, 105, 101, and 1.27, respectively?

I guess I'm not seeing the inconsistency, since I think it is more useful to be consistent in significant digits than in digits. If one F value is 101.something and the other is 1.26 I don't need to know the "something" part.